packages/nodes-base/nodes/Evaluation/Evaluation/Description.node.ts

```diff
@@ -0,0 +1,104 @@
+import type { INodeProperties } from 'n8n-workflow';
+
+import { document, sheet } from '../../Google/Sheet/GoogleSheetsTrigger.node';
+
+export const setOutputProperties: INodeProperties[] = [
+	{
+		displayName: 'Credentials',
+		name: 'credentials',
+		type: 'credentials',
+		default: '',
+	},
+	{
+		...document,
+		displayOptions: {
+			show: {
+				operation: ['setOutput'],
+			},
+		},
+	},
+	{
+		...sheet,
+		displayOptions: {
+			show: {
+				operation: ['setOutput'],
+			},
+		},
+	},
+	{
+		displayName: 'Outputs',
+		name: 'outputs',
+		placeholder: 'Add Output',
+		type: 'fixedCollection',
+		typeOptions: {
+			multipleValueButtonText: 'Add Output',
+			multipleValues: true,
+		},
+		default: {},
+		options: [
+			{
+				displayName: 'Filter',
+				name: 'values',
+				values: [
+					{
+						displayName: 'Name',
+						name: 'outputName',
+						type: 'string',
+						default: '',
+					},
+					{
+						displayName: 'Value',
+						name: 'outputValue',
+						type: 'string',
+						default: '',
+					},
+				],
+			},
+		],
+		displayOptions: {
+			show: {
+				operation: ['setOutput'],
+			},
+		},
+	},
+];
+
+export const setMetricsProperties: INodeProperties[] = [
+	{
+		displayName:
+			"Calculate the score(s) for the evaluation, then map them into this node. They will be displayed in the ‘evaluations’ tab, not the Google Sheet. <a href='https://docs.n8n.io/integrations/builtin/core-nodes/n8n-nodes-base.evaluationmetric/' target='_blank'>View metric examples</a>",
+		name: 'notice',
+		type: 'notice',
+		default: '',
+		displayOptions: {
+			show: {
+				operation: ['setMetrics'],
+			},
+		},
+	},
+	{
+		displayName: 'Metrics to Return',
+		name: 'metrics',
+		type: 'assignmentCollection',
+		default: {
+			assignments: [
+				{
+					name: '',
+					value: '',
+					type: 'number',
+				},
+			],
+		},
+		typeOptions: {
```

Comment:
should it have `multipleValues: true`? because currently only single metric could be added